src/bootstrap/src/core/builder.rs

```diff
@@ -1296,18 +1318,24 @@ impl<'a> Builder<'a> {
         target: TargetSelection,
         cmd: &str,
```

Comment:
nit: You can add the same FIXME of `cmd` arg from `Builder::cargo` just to increase visibility of it.